plugins/source/mixpanel/docs/tables/mixpanel_cohorts.md

```diff
@@ -0,0 +1,24 @@
+# Table: mixpanel_cohorts
+
+The primary key for this table is **id**.
+
+## Relations
+
+The following tables depend on mixpanel_cohorts:
+  - [mixpanel_cohort_members](mixpanel_cohort_members.md)
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|count|Int|
+|is_visible|Int|
```

Comment:
Just asking: should this be a boolean?

Reply:
The API returns an integer and not a boolean. Opted to keep it as-is for parity and code simplicity (would have to go for two separate structs or custom resolvers etc etc otherwise)